codeinputlib/src/main/java/com/github/glomadrian/codeinputlib/CodeInput.java

```diff
@@ -259,8 +265,15 @@ private void reverseAnimation() {
       float toX = sectionpath.getToX() - reduction;
       float toY = sectionpath.getToY();
       drawSection(i, fromX, fromY, toX, toY, canvas);
+      boolean showpass = true;
+      LinearLayout parent = (LinearLayout) this.getParent();
```

Comment:
You imply that every layout using code input will be a linear layout which might not be the case